src/env_utils.py

```diff
@@ -443,6 +443,7 @@ def step(self, action):
             self.episode_return = torch.zeros(1, 1)
             self.episode_step = torch.zeros(1, 1, dtype=torch.int32)
 
+            os.makedirs(pathlib.Path(self.namefile).parent.absolute(), exist_ok=True)
```

Comment:
Code does not automatically create this directory when running the evaluation script on Habitat